Jenkinsfile

```diff
@@ -56,12 +61,18 @@ pipeline {
             buildTestAndScanImage('ubuntu-ruby-fips')
           }
         }
+        stage ('Build, Test, and Scan ubi-ruby-fips image') {
+          steps {
+            buildTestAndScanImageUBI('ubi-ruby-fips')
```

Comment:
The original `buildTestAndScanImage` have test.sh that can't run on the ubi image because it searches for specific outputs for openssl commands and ubuntu specific commands, that is the reason I created a dedicated method that with a comment out of the test phase till we'll have a specific test for ubi.